packages/x-data-grid/src/models/props/DataGridProps.ts

```diff
@@ -290,6 +290,11 @@ export interface DataGridPropsWithDefaultValues {
    * @default "client"
    */
   paginationMode: GridFeatureMode;
+  /**
+   * Set of rows of type [[GridRowsProp]].
+   * @default []
+   */
+  rows?: GridRowsProp<R>;
```

Comment:
```suggestion
  rows: GridRowsProp<R>;
```
Shouldn't be optional? Then in some places, we won't need `props.rows!`.

Reply:
Oh correct 👍 
Thanks for pointing out!